examples/federated-learning/tff/federated-learning-for-image-classification/container-image/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/jupyter/tensorflow-notebook:tensorflow-2.15.0
+
+WORKDIR "/home/jovyan"
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
Refactor to:

```shell
RUN pip install \
    --no-cache-dir \
    --requirement requirements.txt \
    && rm requirements.txt
```

for consistency with other container images.

Reply:
Done

Reply:
Please, use this form with long options and line breaks so it's more readable.